docs/src/main/sphinx/connector/hive.rst

```diff
@@ -433,6 +433,11 @@ Property Name                                      Description
                                                    on auto_purge.
 
 ``hive.partition-projection-enabled``              Enables Athena partition projection support                     ``false``
+
+``hive.iceberg-catalog-name``                      Catalog to redirect to when an Iceberg table is referenced.
```

Comment:
referenced by what? How? maybe

...when an SQL query accesses a table in the catalog that is using the Iceberg table format. The configured catalog needs to be configured on the same cluster and.... 

what else?

Reply:
Use the same metastore? Same credentials? Same storage URL .. ?

Reply:
Do we have a general page about table redirections? If not we should add one. Since other things also need to be mentioned there like how it interacts with DDL vs DML queries and how metadata queries like SHOW or information_schema may be affected.

Reply:
I will do a corresponding PR for table redirection in the context of lakehouse connectors in the beginning of the next week.

Reply:
See https://github.com/trinodb/trino/pull/13773